### PR TITLE
adjusted color styling on careers page

### PR DESCRIPTION
**Pull request by rslaughter12**
changed background color and the background color of the career box

### CODE REVIEW

File: client/src/pages/Careers.js [hunk 1]
```diff
@@ -23,38 +22,38 @@ const useStyles = makeStyles((theme) => ({
...
     marginBottom: theme.spacing(2),
     color: 'white',
     width: '100%',
     '& label': {
-      color: 'white',
+      color: 'black',
     },
     '& input': {
-      color: 'white', // Set the input text color to white
+      color: 'black', // Set the input text color to white
```
Comment (sendusyourbones) [inline client/src/pages/Careers.js]: This comment is now inaccurate since you updated the value to black

File: client/src/pages/Careers.js [hunk 2]
```diff
@@ -23,38 +22,38 @@ const useStyles = makeStyles((theme) => ({
     margin: '0 auto',
     marginTop: theme.spacing(4),
     padding: theme.spacing(3),
-    backgroundColor: 'grey',
+    backgroundColor: '#D8BFD8',
```
Comment (sarah-jensen) [inline client/src/pages/Careers.js]: is this pinkish/purple what you were going for? 
image

Comment (sarah-jensen) [reply client/src/pages/Careers.js]: there's are two blue/gray colors I've been using on the cards for all of the tutorials if you want to use the same. of if there's a different combo we want to switch to, that's fine too. 
Here's what I've been using:
darker: '#92b4d4'
lighter: '#dae9f7'

And I had styled a few buttons that are currently commented out:
button: {
    background: '#f7d148',
    '&:hover': { background: '#f2bf07' },
  },

**Review by sarah-jensen — COMMENTED** [review]
left a couple of comments related to color.

File: client/src/pages/Careers.js (hunk 1 above)
Comment (sarah-jensen) [inline client/src/pages/Careers.js]: The comment is inaccurate since the input text color is black.

File: client/src/pages/Careers.js [hunk 3]
```diff
@@ -23,38 +22,38 @@ const useStyles = makeStyles((theme) => ({
...
+      color: 'black',
     },
     '& input': {
-      color: 'white', // Set the input text color to white
+      color: 'black', // Set the input text color to white
     },
     '& .MuiOutlinedInput-root': {
       '& fieldset': {
-        borderColor: 'white',
+        borderColor: 'black',
```
Comment (sarah-jensen) [inline client/src/pages/Careers.js]: change to gray -- that way the hover you have on line 53 shows.

File: client/src/pages/Careers.js [hunk 4]
```diff
@@ -23,38 +22,38 @@ const useStyles = makeStyles((theme) => ({
...
     display: 'flex',
     flexDirection: 'column',
     alignItems: 'center',
     marginTop: theme.spacing(2),
-    color: 'white',
+    color: 'black',
   },
   inputField: {
     marginBottom: theme.spacing(2),
     color: 'white',
```
Comment (sarah-jensen) [inline client/src/pages/Careers.js:39]: what is this setting the color for?

**Review by sarah-jensen — COMMENTED** [review]
I don't think this was part of this PR, but the spacing on your success message is a bit wonky. The green text is a little weird against the blue background as well.

image

File: client/src/pages/Careers.js (hunk 4 above)
Comment (rslaughter12) [reply client/src/pages/Careers.js:39]: it is doing nothing, lol my b

**Comment by sarah-jensen** [comment]
Success message looks good, I would remove the other text on the page when it pops up though.

**Review by sarah-jensen — APPROVED** [review]

**Pull request merged**